.travis.yml

```diff
@@ -12,8 +11,9 @@ matrix:
       env: TOXENV=py36
     - python: 3.7
       env: TOXENV=py37
-      dist: xenial          # required for Python >= 3.7
-    - python: pypy
+    - python: 3.8
```

Comment:
Please submit another pull request with unrelated changes to CI config.

Reply:
this slipped in by mistake, I forgot this pull request was still open and that I made it from my master branch